guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -32,16 +32,17 @@ The following tables indicate the destination port and the direction of network
 | 69 | UDP | TFTP | Client | TFTP Server (optional) |
 | 443 | TCP | HTTPS | {SmartProxy} | {ProjectName} API | Communication from {SmartProxy}
 ifdef::katello,satellite,orcharhino[]
-| 443, 80 | TCP | HTTPS, HTTP | Client | Content Retrieval | Content
-| 443, 80 | TCP | HTTPS, HTTP | {SmartProxy} | Content Retrieval | Content
-| 443, 80 | TCP | HTTPS, HTTP| Client | Content Host Registration | {SmartProxy} CA RPM installation
 | 443 | TCP | HTTPS | Client | Content Host registration | Initiation
 
 Uploading facts
 
 Sending installed packages and traces
 | 443 | TCP | HTTPS | {ProjectName} | Content Mirroring | Management
 | 443 | TCP | HTTPS | {ProjectName} | {SmartProxy} API | Smart Proxy functionality
+| 443, 80 | TCP | HTTPS, HTTP | {SmartProxy} | Content Retrieval | Content
+| 443, 80 | TCP | HTTPS, HTTP | Client | Content Retrieval | Content
+| 443, 80 | TCP | HTTPS, HTTP| Client | Content Host Registration | Katello CA Consumer RPM
+| 443, 80 | TCP | HTTPS, HTTP| Client | Global Registration | Allow incoming traffic to {Project} for host registration
```

Comment:
```suggestion
| 443, 80 | TCP | HTTPS, HTTP| Client | Global Registration | Registration of hosts to {Project}
```

Reply:
Gobal registration is not a Katello-only feature, but in core Foreman.

Reply:
Addressed both :+1: